cl/citations/match_citations.py

```diff
@@ -218,7 +232,7 @@ def get_citation_matches(
                 for c in cm.cluster.citations.all():
                     if (
                         citation.reporter == c.reporter
-                        and citation.volume == c.volume
+                        and citation.volume == str(c.volume)
```

Comment:
In eyecite volumes are now strings rather than ints (see https://github.com/freelawproject/eyecite/pull/12), so there is some mismatch between eyecite's citation abstraction and CL's. One solution is to migrate all the existing citations in CL from int to str, but that seems dangerous to me and would at least require a separate PR. In the meantime, I simply decided to coerce the volume to str here to do a proper comparison.